amy/workshops/migrations/0269_alter_trainingrequest_default_for_text_fields.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.20 on 2023-10-30 09:20
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    replaces = [('workshops', '0269_alter_trainingrequest_nonprofit_teaching_experience'), ('workshops', '0270_auto_20231030_0919')]
```

Comment:
What causes this? Can you use a rebased codebase or alter migration `0269_alter_trainingrequest_nonprofit_teaching_experience` (it's okay before we release to prod, and even safer if it wasn't released on test-amy)?

Reply:
I squashed two uncommitted migrations to make this one, and this line resulted from that. I'll delete it.